sabnzbd/decoder.py

```diff
@@ -102,7 +102,8 @@ def run(self):
                     if nzo.precheck:
                         raise BadYenc
                     register = True
-                    logging.debug("Decoding %s", art_id)
```

Comment:
Why this one?
I always use the decoding messages as a sign that things go well.
Otherwise nothing is logged.